### PR TITLE
lazydocker: add page

### PR DESCRIPTION
**Pull request by Abhishek-1804**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x]  The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Abhishek-1804** [comment]
@sebastiaanspeck @acuteenvy Hi, Could you please review this pr?

File: pages/common/lazydocker.md
Comment (spageektti) [inline pages/common/lazydocker.md:28]:
```suggestion
- Specify an alternate Docker Compose file:

`lazydocker --file {{path/to/docker-compose.yml}}`

- Display help with available flags:

`lazydocker --help`

- Display version:

`lazydocker --version`
```

**Review by spageektti — APPROVED** [review]

File: pages/common/lazydocker.md
Comment (gutjuri) [inline pages/common/lazydocker.md:22]:
```suggestion
- Display help:
```

**Review by gutjuri — APPROVED** [review]

**Comment by Abhishek-1804** [comment]
Hi @gutjuri , All checks have passed, and the required reviews are complete. Could you please merge this PR at your convenience?
Thanks!

**Review by karthik-script — APPROVED** [review]

**Comment by gutjuri** [comment]
> Hi @gutjuri , All checks have passed, and the required reviews are complete. Could you please merge this PR at your convenience? Thanks!

Usually we do not merge immediately after a PR has been approved. Also, please respect that the reviewers and maintainers of this project are volunteers. Thus, please refrain from tagging individual maintainers asking for reviews. All PRs will be reviewed eventuelly :)

**Pull request merged**
